test/rulesets/Base/base.jl

```diff
@@ -115,6 +115,17 @@
         @test extern(dy) == extern(zeros(2, 5) .+ dy)
     end
 
+     @testset "ldexp" begin
+            rng = MersenneTwister(123456)
+            x, Δx, x̄ = 10rand(rng, 3)
+            Δz = rand(rng)
```

Comment:
The use of local random number generators in the tests is a bad habit that needs to be removed from this code base. but for now lets just not add any more of it.
 The global random number generator is fine.
```suggestion
            x, Δx, x̄ = 10rand(3)
            Δz = rand()
```